quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/BasicDatabaseTest.java

```diff
@@ -17,23 +17,27 @@
 
 package org.keycloak.it.storage.database;
 
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 import org.keycloak.it.junit5.extension.CLIResult;
 
 import io.quarkus.test.junit.main.Launch;
 import io.quarkus.test.junit.main.LaunchResult;
 
-public abstract class AbstractStartDabataseTest {
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public abstract class BasicDatabaseTest {
 
     @Test
-    @Launch({ "start-dev" })
+    @Launch({ "start", "--http-enabled=true", "--hostname-strict=false" })
```

Comment:
Have you considered adding the tests for `start` instead of swapping from `start-dev`?

Reply:
Could you please elaborate?

Reply:
Still not sure what you mean ... We have tests for both commands in the distribution. Here we want to check how the `start` behaves with a pre-built image.